src/main/java/uk/nhs/hee/trainee/details/api/PlacementResource.java

```diff
@@ -68,4 +70,30 @@ public PlacementResource(PlacementService service, PlacementMapper mapper) {
         .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Trainee not found."));
     return ResponseEntity.ok(mapper.toDto(entity));
   }
+
+  /**
+   * Delete the placement for the trainee.
+   *
+   * @param traineeTisId    The ID of the trainee to update.
+   * @param placementTisId  The ID of the placement to delete.
+   * @return True if the placement was deleted.
+   */
+  @DeleteMapping("/{traineeTisId}/{placementTisId}")
+  public ResponseEntity<Boolean> deletePlacement(
+      @PathVariable(name = "traineeTisId") String traineeTisId,
+      @PathVariable(name = "placementTisId") String placementTisId) {
+    log.trace("Delete placement with TIS ID {} of trainee with TIS ID {}",
+        placementTisId, traineeTisId);
+    try {
+      boolean foundTraineeAndPlacement =
+          service.deletePlacementForTrainee(traineeTisId, placementTisId);
+      if (!foundTraineeAndPlacement) {
+        throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Trainee or placement not found.");
+      }
+    } catch (IllegalArgumentException | InvalidDataAccessApiUsageException e) {
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, e.getLocalizedMessage());
+      // other exceptions are possible, e.g. DataAccessException if MongoDB is down
```

Comment:
Could you add a TODO here to go back and add global exception handlers for these types of exceptions?

Reply:
Done